dexie/dexie.d.ts

```diff
@@ -306,7 +314,7 @@ declare module Dexie {
 
     interface IndexSpec {
         name: string;
-        keyPath: any;
+        keyPath: any; // string | Array<string>
```

Comment:
It's just due to backward compatibility with old typescript compiler that don't accept multiple types. However, I believe it's soon time to start using the new typescript syntax but then there are several parts of the definition file that could be enhanced by changing from any to explicit types lists.